src/__tests__/pages/Recommender.test.tsx

```diff
@@ -0,0 +1,263 @@
+    }
+})
+
+test('quiz answer dont know EN', () => {
+    const {getAllByText} = renderRecommenderPage();
+    const obj = getAllByText(/Don't know/i);
+    for (let o of obj) {
+        expect(o).toBeInTheDocument();
+    }
+})
+
+test('quiz answer dont know DA', () => {
+    const {getAllByText} = renderRecommenderPage();
+
+    changeLanguage('da');
+
+    const obj = getAllByText(/Ved ikke/i);
+    for (let o of obj) {
+        expect(o).toBeInTheDocument();
+    }
+})
+
+test("guiz inital guestion EN", async ()=>{
+    const wrapper = mountRecommenderPage();
+
+    await runAllPromises()
+    wrapper.update()
+
+    const title = wrapper.find(".title");   
+    expect(title.get(0).props["children"][3]).toBe('Some initial question?');
+})
+
+test("guiz inital guestion DA", async ()=>{
+    const wrapper = mountRecommenderPage();
+
+    changeLanguage("da");
+
+    await runAllPromises()
+    wrapper.update()
+
+    const title = wrapper.find(".title");   
+    expect(title.get(0).props["children"][3]).toBe('Et udgangspunkt spørgsmål?');
+})
+
+
+each([Answer_Enum.YES, Answer_Enum.PROBABLY, Answer_Enum.DONT_KNOW, Answer_Enum.PROBABLY_NOT, Answer_Enum.NO].map((x) => Answer_Enum[x]))
+    .test("Answer buttons DA %s", async (answer) =>{
+        let secondQuestion = {
+            id: 6,
+            en: "second question?",
+            da: "andet spørgsmål?"
+        };
+        mock.onPost("/question/").reply(200, secondQuestion)
+
+        let wrapper = mountRecommenderPage()
+
+        await runAllPromises()
+        wrapper.update()
+
+        changeLanguage("da")
+        wrapper.find("#"+ answer.toLowerCase() +"_btn").simulate("click")
+
+        await runAllPromises()
+        wrapper.update()
+        
+        const title = wrapper.find(".title");   
+        expect(title.get(0).props["children"][3]).toBe('andet spørgsmål?');
+    })
+
+
+each([Answer_Enum.YES, Answer_Enum.PROBABLY, Answer_Enum.DONT_KNOW, Answer_Enum.PROBABLY_NOT, Answer_Enum.NO].map((x) => Answer_Enum[x]))
+    .test("Answer buttons EN %s", async (answer) =>{
+        let secondQuestion = {
+            id: 6,
+            en: "second question?",
+            da: "andet spørgsmål?"
+        };
+        mock.onPost("/question/").reply(200, secondQuestion)
+
+        let wrapper = mountRecommenderPage()
+
+        await runAllPromises()
+        wrapper.update()
+
+        wrapper.find("#"+ answer.toLowerCase() +"_btn").simulate("click")
+
+        await runAllPromises()
+        wrapper.update()
+        
+        const title = wrapper.find(".title");   
+        expect(title.get(0).props["children"][3]).toBe('second question?');
+    })
+
+test("Answer redirects on 20th question", async () =>{
+    let secondQuestion = {
+        id: 6,
+        en: "second question?",
+        da: "andet spørgsmål?"
+    };
+    mock.onPost("/question/").reply(200, secondQuestion)
+
+    let mockAnswers = [] as Answer[]
+    for(let i = 0; i < 19; i++){
```

Comment:
Skal det her hardcodes?

Reply:
:shrug:

Reply:
Kan ikke se nogen måde at forbedre dette. Men jeg er åben for forslag